netty/src/main/java/io/grpc/netty/SendResponseHeadersCommand.java

```diff
@@ -79,6 +80,6 @@ public String toString() {
 
   @Override
   public int hashCode() {
-    return stream.hashCode();
+    return Objects.hashCode(stream, headers, status);
```

Comment:
sorry to be dumb, but I think header is mutable, and shouldn't go in here.  Just status should be.

Reply:
True, I suppose somebody can create a SendResponseHeadersCommand before filling out the content of the headers object.